### PR TITLE
rework: added libs to conda setup

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
Adds the "xarray" and "netcdf4" libraries to the Conda setup script for use by Civet recipes.

### CODE REVIEW

File: scripts/TestHarness/testers/RavenUtils.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def __lookUpPreferredVersion(name):
                ("scipy",__lookUpPreferredVersion("scipy")),
                ("scikit-learn",__lookUpPreferredVersion("sklearn")),
                ("matplotlib",__lookUpPreferredVersion("matplotlib")),
+               ("xarray",""),
+               ("netcdf4",""),
```
Comment (wangcj05) [inline scripts/TestHarness/testers/RavenUtils.py]: do we have a preferred version?

Comment (PaulTalbot-INL) [reply scripts/TestHarness/testers/RavenUtils.py]: We will, but not yet.  These will be added when we add tensorflow and determine versions for the remaining libraries.

Comment (wangcj05) [reply scripts/TestHarness/testers/RavenUtils.py]: ok, this PR is good to me.

**Comment by PaulTalbot-INL** [comment]
I look forward to seeing the results of the linux test in particular.

**Comment by PaulTalbot-INL** [comment]
The Linux test ended with false positive on overall tests, with a runworker exception:
```
framework.testExternalModelReseed......................................................................... OK
runWorker Exception: Error tokenizing data. C error: Expected 4 fields in line 3, saw 5

-------------------------------------------------------------------------------------------------------------
Ran 27 tests in 117.1 seconds
27 passed, 13 skipped, 0 pending, 0 failed
```
Anyone have any idea what this exception might refer to?

**Comment by PaulTalbot-INL** [comment]
Update: I'm getting an ubuntu virtual machine running so I can try and replicate the linux failure.  It appears to be pandas-and-csv related.

**Comment by PaulTalbot-INL** [comment]
I added the explicit separator to the UnorderedCSVDiffer as `sep=','` for the call to `pandas.read_csv()` and this fixed the break in the TestHarness.  This is available for additional review/merging.

Incidentally, this raised an [issue](https://github.com/idaholab/moose/issues/10607) I'm patching for the test harness in MOOSE.

**Comment by wangcj05** [comment]
Everything looks good.

**Pull request merged**